tests/test_status.py

```diff
@@ -6,9 +6,35 @@
 import sys
 
 import generate_status_reference_data as gen
+import pytest
 import signac
 
 
+@pytest.fixture(params=[True, False])
+def hide_progress_bar(request):
+    return request.param
+
+
+@pytest.fixture(params=["thread", "process", "none"])
+def parallelization(request):
+    return request.param
+
+
+def test_hide_progress_bar(hide_progress_bar, parallelization):
```

Comment:
Do we have a corresponding test for progress bars during execution? That behavior was also changed in this PR.

Reply:
That part has been removed.